packages/components/src/components/atoms/BaseCard/stories/Pure.story.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import { defaultStoryTemplate, defaultStoryMetaFactory } from './common';
+
+export const PureExample = defaultStoryTemplate.bind({});
+PureExample.args = {
+  frontFace: <div>Front face</div>,
+  backFace: <div>Back face</div>,
+};
+
+export default defaultStoryMetaFactory({ title: 'Pure' });
```

Comment:
I don't think we need this file, or do we? What is it doing exactly

Reply:
It's a story showing we can use `BaseCard` with whatever you want as face (so if you want to use another type of card).
Doesn't make sense?

I advise you to run locally to see the stories running to get a better feeling of what is doing